src/printer.rs

```diff
@@ -367,15 +367,26 @@ impl<'a> Printer for InteractivePrinter<'a> {
                     match chunk {
                         // ANSI escape passthrough.
                         (text, true) => {
-                            if text.chars().last().map_or(false, |c| c == 'm') {
+                            let is_ansi_csi =
+                                text.chars().skip(1).nth(0).map_or(false, |c| c == '[');
```

Comment:
Could this be replaced by `… = text.starts_with("\x1B[")`?

Reply:
My bad, haha. I'm a bit rusty (pun intended) with the standard library functions.